README.md

```diff
@@ -13,13 +13,29 @@ Aptos is a layer 1 blockchain bringing a paradigm shift to Web3 through better t
 
 ## Getting Started
 
-* [Aptos Labs](https://aptoslabs.com/)
+### Quick
+
+```sh
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+cargo build
+cargo test
+```
+
```

Comment:
```suggestion
```

Reply:
I don't think we need this content duplicated here.

Reply:
(process: You should not "comment close", things disappear easily that way)

its a code repo. Quickstarts are quasi-standard, and slight duplication is never avoidable.